x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -10,10 +10,10 @@ import { useEuiTheme } from '@elastic/eui';
 import { CSSObject } from '@emotion/react';
 
 interface StylesDeps {
-  height: number | undefined;
+  height: string | undefined;
 }
 
-export const useStyles = ({ height = 500 }: StylesDeps) => {
+export const useStyles = ({ height = '500px' }: StylesDeps) => {
```

Comment:
Just changed this to a string, so we aren't constrained to px units. We do need to figure out what to do with this in full screen mode though, as it looks not quite right.

Reply:
yeah, knowing the session view height for full screen shouldn't be other plugins' responsibility, so I suggest we add something like this [PR suggestion](https://github.com/kqualters-elastic/kibana/pull/5) later